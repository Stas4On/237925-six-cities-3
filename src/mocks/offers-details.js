export const offersDetailsMock = [
  {
    name: `Beautiful &amp; luxurious apartment at great location`,
    price: 120,
    rating: 4,
    type: `apartment`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`],
    bedrooms: 2,
    guest: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: true,
    id: 12
  },
  {
    name: `Wood and stone place`,
    price: 80,
    rating: 3.5,
    type: `private room`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`],
    bedrooms: 2,
    guest: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: false,
    id: 31
  },
  {
    name: `Canal View Prinsengracht`,
    price: 132,
    rating: 2.33,
    type: `apartment`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`],
    bedrooms: 2,
    guest: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: false,
    id: 23
  },
  {
    name: `Nice, cozy, warm big bed apartment`,
    price: 180,
    rating: 4.5,
    type: `apartment`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`, `img/room.jpg`],
    bedrooms: 2,
    guest: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: true,
    id: 1
  }
];
