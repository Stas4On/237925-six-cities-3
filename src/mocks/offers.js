export const numberRentalOffers = 56;
export const offers = [
  {
    name: `Beautiful &amp; luxurious apartment at great location`,
    price: 120,
    rating: 4,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    coordinates: [52.3909553943508, 4.85309666406198],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: true,
    id: 12
  },
  {
    name: `Wood and stone place`,
    price: 80,
    rating: 3.5,
    type: `private room`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    coordinates: [52.369553943508, 4.85309666406198],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: false,
    id: 31
  },
  {
    name: `Canal View Prinsengracht`,
    price: 132,
    rating: 2.33,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    coordinates: [52.3909553943508, 4.929309666406198],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: false,
    id: 23
  },
  {
    name: `Nice, cozy, warm big bed apartment`,
    price: 180,
    rating: 4.5,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    coordinates: [52.3809553943508, 4.939309666406198],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: true,
    id: 1
  }
];
