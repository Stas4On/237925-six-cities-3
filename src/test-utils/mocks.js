export const offerCards = [
  {
    city: {
      name: `Paris`,
      location: {
        latitude: 52.37454,
        longitude: 4.897976,
        zoom: 13
      }
    },
    name: `Beautiful apartment`,
    price: 150,
    rating: 3,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: false,
    id: 34
  },
  {
    city: {
      name: `Hamburg`,
      location: {
        latitude: 52.37454,
        longitude: 4.897976,
        zoom: 13
      }
    },
    name: `Wood and stone place`,
    price: 550,
    rating: 1,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: true,
    premium: true,
    id: 3
  },
  {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.37454,
        longitude: 4.897976,
        zoom: 13
      }
    },
    name: `Canal View Prinsengracht`,
    price: 132,
    rating: 2.33,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: false,
    premium: false,
    id: 23
  },
  {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.37454,
        longitude: 4.897976,
        zoom: 13
      }
    },
    name: `Warm apartment`,
    price: 20,
    rating: 5,
    type: `apartment`,
    imgUrl: `img/room.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`,
    features: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Coffee machine`, `Baby seat`, `Kitchen`, `Dishwasher`, `Cabel TV`, `Fridge`],
    photos: [`img/room.jpg`, `img/apartment-03.jpg`, `img/room.jpg`, `img/apartment-02.jpg`, `img/room.jpg`, `img/apartment-01.jpg`],
    bedrooms: 2,
    guests: 4,
    owner: `Angelina`,
    avatar: `img/avatar-angelina.jpg`,
    super: true,
    favorite: true,
    premium: true,
    id: 67
  }
];

export const cities = [
  `Paris`,
  `Cologne`,
  `Brussels`,
  `Amsterdam`,
  `Hamburg`,
  `Dusseldorf`
];

export const initialCity = `Paris`;

export const numberRentalOffers = 12;
