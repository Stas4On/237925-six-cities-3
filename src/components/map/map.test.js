
import React from 'react';
import renderer from 'react-test-renderer';

import Map from './map';
import {offerCards} from "@/test-utils/mocks";

it(`should render correctly Map component`, () => {
  const MapComponent = renderer.create(<Map offers={offerCards}/>).toJSON();

  expect(MapComponent).toMatchSnapshot();
});
