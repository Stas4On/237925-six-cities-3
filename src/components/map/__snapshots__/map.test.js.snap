// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly Map component 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
